package.json

```diff
@@ -38,7 +38,7 @@
     "localchaindb:build": "yarn localchaindb:clean && ./scripts/runganache.sh --db ./localchaindb & yarn migrate && echo 'Migration done. Ganache db in ./localchaindb folder.'",
     "localchaindb:builddocker": "yarn localchaindb:build && yarn docker:build",
     "docker:build": "docker build . -t localdockerimage",
-    "docker:run": "docker rm ganache ; docker run --init --name ganache -p 8545:8545 localdockerimage --db ./dockerLocalchaindb --gasLimit 0x47D5DE --gasPrice 1000000000 --networkId 999 -m \"hello build tongue rack parade express shine salute glare rate spice stock\" & wait",
+    "docker:run": "docker rm ganache ; docker run --init --name ganache -p 8545:8545 localdockerimage --db ./dockerLocalchaindb --gasLimit 0xFFFFFF --gasPrice 1000000000 --networkId 999 -m \"hello build tongue rack parade express shine salute glare rate spice stock\" & wait",
```

Comment:
setting the gasLimit so high can cause troubles because we will only see if contract too big when we try to deploy to Rinkeby/mainnet. I would set ganache gasLimit to the lower of RInkeby/mainnet block limit (and even a bit lower as blocklimit can shrink too)

Reply:
Block limit is not constant, currently it is around 8 million on mainnet, and 7 million on rinkeby. Apart from small fluctuations it seems to be constantly growing (slowly).
The previous value of 4.7 million caused serious problems on the local ganache test network, so we had to increase it. We did not yet think about the correct final value, just increase it to get this obstacle out of the way, so we can make progress with the actual feature.
Setting it slightly below the lower of the above actual blocklimits seems sensible, i will set it to 6.9 million then.